conf/test_star.config

```diff
@@ -0,0 +1,30 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ *   nextflow run nf-core/scrnaseq -profile test
+ */
+
+params {
```

Comment:
This file looks very similar to `test.config`.
Did you try simplifying this one and including it?

Reply:
Now I did ;-)